cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -893,6 +893,14 @@
     </description>
   </property>
 
+  <property>
+    <name>data.storage.sql.scan.size</name>
```

Comment:
Be specific on what the unit of the size, bytes? number of rows? or something else?

Reply:
Done